Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/InAppNotification.xaml

```diff
@@ -34,4 +30,19 @@
         <Setter Property="HorizontalOffset" Value="0" />
         <Setter Property="Template" Value="{StaticResource MSEdgeNotificationTemplate}" />
     </Style>
+
+    <contract5NotPresent:Style TargetType="local:InAppNotification" BasedOn="{StaticResource BaseInAppNotificationsStyle}">
+        <Setter Property="Background" Value="{ThemeResource SystemControlBackgroundChromeMediumBrush}" />
+    </contract5NotPresent:Style>
+
+    <contract5Present:Style TargetType="local:InAppNotification" BasedOn="{StaticResource BaseInAppNotificationsStyle}">
+        <Setter Property="Background">
+            <Setter.Value>
+                <AcrylicBrush BackgroundSource="Backdrop" 
```

Comment:
Does this also need the contract present or is that inherited from the style?

Reply:
It isn't even parsed because it is enclosed in the Conditional XAML check. 

"Conditional statements are evaluated at runtime, and elements qualified with a conditional XAML tag are parsed if they evaluate to true; otherwise, they are ignored."

https://docs.microsoft.com/en-us/windows/uwp/debug-test-perf/conditional-xaml